docs/pages/dev-guide/developer-guide.md

```diff
@@ -38,6 +38,69 @@ Although all SDK services written today store metadata in Zookeeper, this is an
 
 They store the desired configuration of a service and all relevant information regarding Mesos tasks, respectively, but the precise format or location of the underlying data may be customized.  For example, the data may be stored in Zookeeper, but in a different format, or the data may be stored in a different persistent storage like etcd.  The defaults should be reasonable for most developers, however. Support for optional customization via drop-in replacement is a common pattern throughout the SDK.
 
+# Getting Started
+
+1. Create your framework
+
+   ```bash
+   $ ./new-service.sh frameworks/myframework
+   $ cd frameworks/myframework
+   ```
+
+   `new-service.sh` creates a skeleton framework.  You will extend
+   this skeleton.
+
+1. View `svc.yml`
+
+   Take a look at `src/main/dist/svc.yml`.  This is the YAML file that
+   defines your framework.  You will be editing this file.
+
+1. View `Main.java`
+
+   Take a look at
+   `src/main/java/com/mesosphere/sdk/mgummelt/scheduler/Main.java`.
```

Comment:
In the future, we should probably move this file out of a directory named after you.